docs/dev_docs/adding_to_the_documentation.rst

```diff
@@ -0,0 +1,21 @@
+Adding to the documentation
+===========================
+
+There is a multitude of ways you can add to the documentation.
+
+Building the docs
```

Comment:
Would it make sense to merge the autodoc_generate and build_docs commands, so that only a single commad has to be executed to build the docs? Having two commands in "build docs" but then calling only one of them "build docs" might be confusing.

Reply:
changed it.